### ISSUE TITLE
PRO - contributor

### ISSUE THREAD

**Issue by prk-dismoi** [dis-moi/contribution-interface#3]

**Comment by prk-dismoi** [comment]
![image](https://user-images.githubusercontent.com/80406294/112133771-51fc2380-8bcc-11eb-922a-738b8f1e7ce9.png)

The contributor form should allow users to:

- add a banner image
- add an avatar image
- add a contributor account name (mandatory)
- add a website (optional)
- choose a category (optional)
- offer option for user to visit their public profile once account was created

**Comment by prk-dismoi** [comment]
Trello design ticket: https://trello.com/c/uZOj15QQ/275-create-contributor-account

**Comment by newick** [comment]
@prk-dismoi I think one missed the case of the used name.

**Comment by prk-dismoi** [comment]
yep you right. Can you slap an error message somewhere? will amend this afternoon.

**Comment by newick** [comment]
![image](https://user-images.githubusercontent.com/657454/113296828-3db9d400-92fa-11eb-9690-11a43ee924c3.png)
WIP
